### ISSUE TITLE
Creating column span using Write(value, style, span) method produces invalid xlsx unless SkipColumns(n - 2) called

### ISSUE THREAD

**Issue by LittleNetworkHack** [salvois/LargeXlsx#41]
Task:
I want to have multi column spanned headers for table

The code for reproducing bug:
```
using (var stream = new FileStream(reportOutputFilePath, FileMode.Create, FileAccess.Write))
using (var writer = new XlsxWriter(stream, requireCellReferences: false))
{
    writer.BeginWorksheet("Test").BeginRow();
    writer.Write("first", XlsxStyle.Default, 3);
    writer.Write("Second", XlsxStyle.Default, 4);
}
```

Problem:
Excel complains with error:
"We found a problem with some content in "output2.xlsx". Do you want us to try to recover as much as we can? If you trust.. click Yes"

If I click "Yes" here is what "fixed" looks like:
![image](https://github.com/salvois/LargeXlsx/assets/45490363/cee5fd23-8a44-493a-8509-ea5a1d2af47b)

Things of note:
It requires me to set `requireCellReferences = false`, else i get same error no matter what

Solution:
```
using (var stream = new FileStream(reportOutputFilePath, FileMode.Create, FileAccess.Write))
using (var writer = new XlsxWriter(stream, requireCellReferences: false))
{
    writer.BeginWorksheet("Test").BeginRow();
    writer.Write("first", XlsxStyle.Default, 3).SkipColumns(1);
    writer.Write("Second", XlsxStyle.Default, 4).SkipColumns(1); // This trailing one can be anything since we don't write after it
}
```

The value of skipped columns must be columnSpan - 2 for it to work.
Skipping columns less than N-2 causes error, skipping more, works as intended (moving insertion point forward)

Can someone help me with this, is this bug? is this normal? am I doing it wrong? is the new excel just picky?

Also here is XML of the error:
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
    <logFileName>error320960_08.xml</logFileName>
    <summary>Errors were detected in file 'C:\Workspace\output2.xlsx'</summary>
    <removedRecords>
        <removedRecord>Removed Records: Cell information from /xl/worksheets/sheet1.xml part</removedRecord>
        <removedRecord>Removed Records: Merge cells from /xl/worksheets/sheet1.xml part</removedRecord>
    </removedRecords>
</recoveryLog>
```

**Comment by salvois** [comment]
Hi @LittleNetworkHack ,
this seems related to issue #40, which has just been solved in release 1.9.1.
From a quick check your code snippet works for me.
Please let me know whether this solves your problem.
Thanks,
   Salvo

**Comment by LittleNetworkHack** [comment]
It works now, thank you. I'll be closing this.
